packages/components/src/components/buttons/buttonStyleUtils.ts

```diff
@@ -120,6 +160,7 @@ export const useVariantStyle = (
             textSubtle: theme.textAlertRed,
         },
     };
+    console.log('___', variantsColors[variant]);
```

Comment:
forgotten console.log?

Reply:
Resolved in 0f791f73ff955c613dff25ba0eaebd92e7955470 
@jvaclavik please squash commits before merge